lib/puppet/defaults.rb

```diff
@@ -1570,7 +1570,10 @@ def self.default_diffargs
     :pluginsync => {
       :default    => true,
       :type       => :boolean,
-      :desc       => "Whether plugins should be synced with the central server.",
+      :desc       => "Whether plugins should be synced with the central server. This setting is
```

Comment:
Thanks, @joshcooper this makes more sense. I think it's still important to notate the behavior if a cached catalog is being used? I think this language is okay as is. :+1: